### PR TITLE
govc: Add version 0.22.1

### LINKED ISSUE #739: [Request] Add govc
Please add support for govc ("govc is a vSphere CLI built on top of govmomi."). See: https://github.com/vmware/govmomi/releases

### PR DESCRIPTION
**Pull request by p1gsy**
- Closes #739 

Tested for installation, uninstallation, and updates

### CODE REVIEW

File: bucket/govc.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://github.com/vmware/govmomi",
+    "description": "A vSphere CLI built on top of govmomi.",
+    "version": "0.22.1",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "bin": [
```
Comment (Ash258) [inline bucket/govc.json]:
```suggestion
```

File: bucket/govc.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                [
+                    "govc_windows_amd64.exe",
+                    "govc"
+                ]
+            ],
+            "url": "https://github.com/vmware/govmomi/releases/download/v0.22.1/govc_windows_amd64.exe.zip",
+            "hash": "081b0c66b82d5d633200881ba03783dd7973b9c89a81acba620167b7f9256d86"
```
Comment (Ash258) [inline bucket/govc.json]:
```suggestion
            "hash": "081b0c66b82d5d633200881ba03783dd7973b9c89a81acba620167b7f9256d86",
            "bin": [
                [
                    "govc_windows_amd64.exe",
                    "govc"
                ]
            ],
```

File: bucket/govc.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        },
+        "32bit": {
+            "bin": [
```
Comment (Ash258) [inline bucket/govc.json]:
```suggestion
```

File: bucket/govc.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+                [
+                    "govc_windows_386.exe",
+                    "govc"
+                ]
+            ],
+            "url": "https://github.com/vmware/govmomi/releases/download/v0.22.1/govc_windows_386.exe.zip",
+            "hash": "031cb9cd22344610a24dd74940cf96369324e17f1d7099a14025469c74f54d9a"
```
Comment (Ash258) [inline bucket/govc.json]:
```suggestion
            "hash": "031cb9cd22344610a24dd74940cf96369324e17f1d7099a14025469c74f54d9a",
            "bin": [
                [
                    "govc_windows_386.exe",
                    "govc"
                ]
            ],
```

File: bucket/govc.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        }
+    },
+    "checkver": "github",
```
Comment (Ash258) [inline bucket/govc.json]: Alternatively use only this:

```suggestion
    "pre_install": "Get-ChildItem \"$dir\\govc_*exe\" | Rename-Item -NewName 'govc.exe'",
    "bin": "govc.exe",
    "checkver": "github",
```

File: bucket/govc.json [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/vmware/govmomi/releases/download/v$version/govc_windows_amd64.exe.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/vmware/govmomi/releases/download/v$version/govc_windows_386.exe.zip"
+            }
+        }
```
Comment (Ash258) [inline bucket/govc.json]:
```suggestion
        },
        "hash": {
            "url": "https://github.com/vmware/govmomi/releases/tag/v$version"
        }
```

File: bucket/govc.json (hunk 5 above)
Comment (p1gsy) [reply bucket/govc.json]: In my view, it's better keep the original name of the program

**Pull request merged**